### PR TITLE
Pure bootstrap+rename from sample-controller

### PR DESCRIPTION
**Pull request by n3wscott**
This is a clone of knative-sandbox/sample-controller with several renames.

### CODE REVIEW

File: pkg/reconciler/ducktype/controller.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	"context"
+
+	"knative.dev/pkg/tracker"
+
+	corev1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/controller"
+	"knative.dev/pkg/logging"
+
+	svcinformer "knative.dev/pkg/client/injection/kube/informers/core/v1/service"
```
Comment (mattmoor) [inline pkg/reconciler/ducktype/controller.go:29]: Format Go code:
```suggestion

```

File: pkg/reconciler/ducktype/controller.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	ducktypeinformer "knative.dev/discovery/pkg/client/injection/informers/discovery/v1alpha1/ducktype"
+	ducktypereconciler "knative.dev/discovery/pkg/client/injection/reconciler/discovery/v1alpha1/ducktype"
```
Comment (mattmoor) [inline pkg/reconciler/ducktype/controller.go:31]: Format Go code:
```suggestion
	ducktypereconciler "knative.dev/discovery/pkg/client/injection/reconciler/discovery/v1alpha1/ducktype"
	svcinformer "knative.dev/pkg/client/injection/kube/informers/core/v1/service"
```

File: pkg/reconciler/ducktype/ducktype.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+package ducktype
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+
```
Comment (mattmoor) [inline pkg/reconciler/ducktype/ducktype.go:25]: Format Go code:
```suggestion

	samplesv1alpha1 "knative.dev/discovery/pkg/apis/discovery/v1alpha1"
	ducktypereconciler "knative.dev/discovery/pkg/client/injection/reconciler/discovery/v1alpha1/ducktype"
```

File: pkg/reconciler/ducktype/ducktype.go [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+	"knative.dev/pkg/logging"
+	"knative.dev/pkg/network"
+	"knative.dev/pkg/reconciler"
+	"knative.dev/pkg/tracker"
+	samplesv1alpha1 "knative.dev/discovery/pkg/apis/discovery/v1alpha1"
+	ducktypereconciler "knative.dev/discovery/pkg/client/injection/reconciler/discovery/v1alpha1/ducktype"
```
Comment (mattmoor) [inline pkg/reconciler/ducktype/ducktype.go:33]: Format Go code:
```suggestion

```

**Comment by lberk** [comment]
/hold
(for linting)

**Pull request merged**